labels.yml

```diff
@@ -77,32 +77,95 @@ cloud-native-taiwan:
     repositories:
     - meetups
   # Jobs board labels
-  - name: Senior
-    color: 008672
+  - name: Kubernetes
+    color: #0075ca
```

Comment:
We don't need the prefix of '#' in the color field. Plz, remove it.

Reply:
Remove the prefix of '#'.